spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -80,6 +82,21 @@ def expect_retryable_error(error_class)
           end.not_to change(Sidekiq::Form526BackupSubmissionProcess::Submit.jobs, :size)
           expect(Form526JobStatus.last.status).to eq 'success'
         end
+
+        it 'updates Form526Submission form with id' do
+          expect(described_class).to be < EVSS::DisabilityCompensationForm::SubmitForm526
+          subject.perform_async(submission.id)
+
+          expect do
+            VCR.use_cassette('virtual_regional_office/contention_classification') do
```

Comment:
More for my own curiosity due to discussions that have happened recently; do you know how this cassette was generated? Was it recorded from an actual request made?

Reply:
Awesome. I had a suspicion some of the unit tests in our test suite were not set up to run actual requests properly, and were relying on copied cassettes. Was just curious, but good to know that doesn't seem to be the case here

Reply:
I believe you could also re-record a cassette by removing the contents of an existing one, though that may not be the case anymore